test/e2e/lib/pages/my-home-page.js

```diff
@@ -58,9 +56,14 @@ export default class MyHomePage extends AsyncBaseContainer {
 		return assert( badgeText === 'Complete', 'Could not locate message that email is verified.' );
 	}
 
-	async isSiteLaunched() {
-		await this.closeCelebrateNotice();
-		return await driverHelper.isElementLocated( this.driver, this.launchSiteTaskCompleteSelector );
+	async waitForSiteLaunchComplete() {
+		await driverHelper.waitUntilElementLocated( this.driver, this.celebrateNoticeTitleSelector );
+		const notice = await this.driver.findElement( this.celebrateNoticeTitleSelector );
+		const noticeText = await notice.getText();
+		return assert(
+			noticeText === 'You launched your site!',
```

Comment:
Would it be ok just to assert that the notice exists? It seems more brittle to hardcode the text!

Reply:
I wasn't sure if maybe there were different possible titles that might be displayed (like maybe "sorry that didn't work").

I suppose just validating that a title exists at all is good but I think that the text matching actually is a benefit for verifying the launch worked. It's like the [testing-library philosophy](https://testing-library.com/docs/react-testing-library/intro): always test what the user sees, not the CSS. If someone changes the text, updating this test should be quite easy. 

Anyway, that was my thought but I'm certainly willing to be convinced otherwise.

Reply:
yeah that's a fair point!